src/install/mod.rs

```diff
@@ -199,15 +194,14 @@ fn prebuilt_url(tool: &Tool, version: &str) -> Result<String, failure::Error> {
         Tool::CargoGenerate => {
             Ok(format!(
                 "https://github.com/cargo-generate/cargo-generate/releases/download/v{0}/cargo-generate-v{0}-{1}.tar.gz",
-                // Krate::new(&Tool::CargoGenerate)?.max_version,
-                "0.5.1", // latest released binary [#907](https://github.com/rustwasm/wasm-pack/issues/907)
+                "0.17.3",
                 target
             ))
         },
         Tool::WasmOpt => {
             Ok(format!(
         "https://github.com/WebAssembly/binaryen/releases/download/{vers}/binaryen-{vers}-{target}.tar.gz",
-        vers = "version_108",
+        vers = "version_110",
```

Comment:
111 is out, looks like not a big change. It would be great to bump to that, and hopefully no more past that and this is merged soon :crossed_fingers:.

https://github.com/WebAssembly/binaryen/releases/tag/version_111
https://github.com/WebAssembly/binaryen/releases/tag/version_111

Reply:
fixed